perses/tests/utils.py

```diff
@@ -410,18 +419,30 @@ def  generate_solvated_hybrid_test_topology(current_mol_name="naphthalene", prop
     from openmoltools import forcefield_generators
 
     from openeye import oechem
-    from openmoltools.openeye import iupac_to_oemol, generate_conformers
+    from openmoltools.openeye import iupac_to_oemol, generate_conformers, smiles_to_oemol
     from openmoltools import forcefield_generators
     import perses.utils.openeye as openeye
     from perses.utils.data import get_data_filename
     from perses.rjmc.topology_proposal import TopologyProposal, SystemGenerator, SmallMoleculeSetProposalEngine
     import simtk.unit as unit
     from perses.rjmc.geometry import FFAllAngleGeometryEngine
 
-    old_oemol, new_oemol = iupac_to_oemol(current_mol_name), iupac_to_oemol(proposed_mol_name)
-
-    old_smiles = oechem.OECreateSmiString(old_oemol,oechem.OESMILESFlag_DEFAULT | oechem.OESMILESFlag_Hydrogens)
-    new_smiles = oechem.OECreateSmiString(new_oemol,oechem.OESMILESFlag_DEFAULT | oechem.OESMILESFlag_Hydrogens)
+    if current_mol_name != None and proposed_mol_name != None:
+        try:
+            old_oemol, new_oemol = iupac_to_oemol(current_mol_name), iupac_to_oemol(proposed_mol_name)
+            old_smiles = oechem.OECreateSmiString(old_oemol,oechem.OESMILESFlag_DEFAULT | oechem.OESMILESFlag_Hydrogens)
```

Comment:
Perhaps you want to move the SMILES string creation options (`oechem.OESMILESFlag_DEFAULT | oechem.OESMILESFlag_Hydrogens`) to a single global definition? Otherwise, you risk getting things out of sync.